wikipedia_scraper/settings.py

```diff
@@ -86,3 +86,13 @@
 #HTTPCACHE_DIR = 'httpcache'
 #HTTPCACHE_IGNORE_HTTP_CODES = []
 #HTTPCACHE_STORAGE = 'scrapy.extensions.httpcache.FilesystemCacheStorage'
+
+#Mongo config
+ITEM_PIPELINES = {'wikipedia_scraper.pipelines.MongoDBPipeline': 100}
+MONGODB_SERVER = "localhost"
```

Comment:
The server and port can be kept together in a single mongodb:// url. This is currently used in main